Bugfix/consolidate exch accounting book (#36)
fivetran/dbt_netsuite_source: CHANGELOG.md

# dbt_netsuite_source v0.7.0
## 🚨 Breaking Changes 🚨:

> - Brought in `accounting_book_id` (accountingbook) to the `stg_netsuite2__consolidated_exchange_rates` model. This will cause v0.8.1 of the [dbt_netsuite package](https://github.com/fivetran/dbt_netsuite) to fail, so make sure to update to v0.9.0. ([PR #36](https://github.com/fivetran/dbt_netsuite_source/pull/36)) 

Reviewer: Since we are forcing the dependency in v0.9.0 of dbt_netsuite I don't believe this is entirely necessary to callout. Instead I would just call out that due to code logic in v0.8.1, this update will lead to an ambiguous column error. Therefore this is treated as a breaking change.
```suggestion
- Brought in `accounting_book_id` (accountingbook) to the `stg_netsuite2__consolidated_exchange_rates` model. ([PR #36](https://github.com/fivetran/dbt_netsuite_source/pull/36)) 
   - Due to logic within the dbt_netsuite v0.8.1 package version, this update will result in an ambiguous columns error. Therefore, this update is treated as a breaking change.
```

Author: Thanks for explaining, makes sense to me

# dbt_netsuite_source v0.6.4

> 

Reviewer: Spacing suggestion
```suggestion
```

## 🎉 Feature Updates 🎉
- Brought in `accounting_book_id` (accountingbook) to the `stg_netsuite2__consolidated_exchange_rates` model. ([PR #36](https://github.com/fivetran/dbt_netsuite_source/pull/36))

# dbt_netsuite_source v0.6.3

> ## Under the Hood:

Reviewer: I think this is an artifact of a merge conflict. This should probably be removed.

```suggestion
```
